### ISSUE TITLE
How to improve the speed of writing?

### ISSUE THREAD

**Issue by zhyhang** [pilosa/pilosa#1556]
- Pilosa 1.0.1,
- **single node**,
- **pcie ssd 2T, 128GB memory, 24ht, CentOS 7.1** 
- java-client
- one string keys index and one string keys field (default options)
- 80k set cmds per request
- every column about set 10 rows
- now total column about 400 millions
- 20 threads
- We only can achieve 1700 columns setting per second.
- **We see the cpu only using one core, and ssd iops is normal.**

**Comment by jaffee** [comment]
@zhyhang using `Set(...)` commands through the `/index/name/query` endpoint is going to be very slow. We have a `/import` endpoint specifically for doing data ingest which is much faster.

Unfortunately, the java client does not yet support imports with string keys, but luckily, @yuce is working on implementing that. We'll let you know here when that work is complete.

**Comment by zhyhang** [comment]
@jaffee Thanks.

**Comment by zhyhang** [comment]
@jaffee Is the API already support string keys import?

**Comment by jaffee** [comment]
Yes, pilosa supports it. It is a bit rough currently, though. You have to import to the primary translator node.
